docs-client/src/containers/MethodPage/EndpointPath.tsx

```diff
@@ -37,15 +41,25 @@ const EndpointPath: React.FunctionComponent<Props> = (props) => (
     <Typography variant="body2" paragraph />
     {props.endpointPathOpen && (
       <>
+        {props.examplePaths.length > 0 && (
+          <>
+            <Typography variant="body2" paragraph />
+            <Dropdown
+              placeholder="Select an example path..."
+              options={props.examplePaths}
+              onChange={props.onSelectedPathChange}
+            />
+          </>
+        )}
+        <Typography variant="body2" paragraph />
         <TextField
-          multiline
           fullWidth
-          rows={1}
-          value={props.endpointPath}
+          value={props.additionalPath}
           placeholder={endpointPathPlaceHolder}
-          onChange={props.onEndpointPathChange}
+          onChange={props.onPathFormChange}
           inputProps={{
-            className: 'code',
+            readOnly: !props.editable,
+            className: 'code ',
```

Comment:
Out of curiosity: is there special meaning for a space after `code`?

Reply:
Oops... it's my mistake. thanks for catching it!